stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -993,6 +999,47 @@ public func _tffunc<State : _TensorArrayProtocolEnhanced,
   }
 }
 
+// Trace the given function to generate a TF graph and return a closure
+// that can be used to launch the graph.
+public func _graph<In : TensorGroup, Out : TensorGroup>(
+  _ fn: (In) -> Out, useXla: Bool = false
+) -> (In) -> Out {
+  let traceContext: TraceContext = withoutActuallyEscaping(fn) { escapableFn in
+    let wrappedFn = {
+      (inputs: [CTensorHandle]) -> [CTensorHandle] in
```

Comment:
I think this could be moved to the previous line without going over the column limit.